python/pyspark/ml/feature.py

```diff
@@ -931,6 +936,64 @@ class StringIndexerModel(JavaModel):
     """
     Model fitted by StringIndexer.
     """
+    @property
+    def labels(self):
+        return self._java_obj.labels
+
+
+@inherit_doc
+class IndexToString(JavaTransformer, HasInputCol, HasOutputCol):
+    """
+    .. note:: Experimental
+    A [[Transformer]] that maps a column of string indices back to a new column of corresponding
+    string values using either the ML attributes of the input column, or if provided using the
+    labels supplied by the user.
+    All original columns are kept during transformation.
+    See L{StringIndexer} for converting strings into indices
+    """
+
+    # a placeholder to make the labels show up in generated doc
+    labels = Param(Params._dummy(), "labels",
+                   "Optional labels to be provided by the user, if not supplied column " +
+                   "metadata is read for labels. The default value is an empty array, " +
+                   "but the empty array is ignored and column metadata used instead.")
```

Comment:
nit: this doc seems redundant IMO, but it's a matter of taste so feel free to keep or leave